doc/source/notebooks/theory/Sanity_check.ipynb

```diff
@@ -67,61 +72,27 @@
-     "text": [
-      "WARNING:tensorflow:From /home/st/anaconda3/envs/relaxedgp/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.\n",
-      "Instructions for updating:\n",
-      "Colocations handled automatically by placer.\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:From /home/st/anaconda3/envs/relaxedgp/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.\n",
-      "Instructions for updating:\n",
-      "Colocations handled automatically by placer.\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:From /home/st/anaconda3/envs/relaxedgp/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3528: setdiff1d (from tensorflow.python.ops.array_ops) is deprecated and will be removed after 2018-11-30.\n",
-      "Instructions for updating:\n",
-      "This op will be removed after the deprecation date. Please switch to tf.sets.difference().\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:From /home/st/anaconda3/envs/relaxedgp/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3528: setdiff1d (from tensorflow.python.ops.array_ops) is deprecated and will be removed after 2018-11-30.\n",
-      "Instructions for updating:\n",
-      "This op will be removed after the deprecation date. Please switch to tf.sets.difference().\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "m1 = gpflow.models.GPR(X, Y, kern=gpflow.kernels.RBF(1))\n",
-    "m2 = gpflow.models.VGP(X, Y, gpflow.kernels.RBF(1), likelihood=gpflow.likelihoods.Gaussian())\n",
-    "m3 = gpflow.models.SVGP(X, Y, gpflow.kernels.RBF(1),\n",
-    "                      likelihood=gpflow.likelihoods.Gaussian(),\n",
-    "                      Z=X.copy(), q_diag=False)\n",
-    "m3.feature.set_trainable(False)\n",
-    "m4 = gpflow.models.SVGP(X, Y, gpflow.kernels.RBF(1),\n",
-    "                      likelihood=gpflow.likelihoods.Gaussian(),\n",
-    "                      Z=X.copy(), q_diag=False, whiten=True)\n",
-    "m4.feature.set_trainable(False)\n",
-    "m5 = gpflow.models.SGPR(X, Y, gpflow.kernels.RBF(1), Z=X.copy())\n",
-    "m5.feature.set_trainable(False)\n",
-    "m6 = gpflow.models.GPRFITC(X, Y, gpflow.kernels.RBF(1), Z=X.copy())\n",
-    "m6.feature.set_trainable(False)\n",
+    "data = (tf.convert_to_tensor(X, dtype=default_float()), tf.convert_to_tensor(Y, dtype=default_float()))\n",
+    "inducing_variable = tf.convert_to_tensor(X, dtype=default_float())\n",
+    "\n",
+    "m1 = gpflow.models.GPR(data, kernel=gpflow.kernels.RBF())\n",
```

Comment:
would be good to take this opportunity to rename RBF -> SquaredExponential ..:)

Reply:
ok